addons/mail/static/src/components/attachment/attachment.xml

```diff
@@ -82,18 +82,20 @@
                                 <i class="fa fa-spin fa-spinner"/>
                             </div>
                         </t>
+                        <!-- Remove button -->
+                        <t t-if="props.isEditable">
+                            <div class="o_Attachment_asideItem" t-att-class="{ 'o-pretty': attachment.isLinkedToComposer }">
+                                <div class="o_Attachment_asideItemUnlink" t-on-click="_onClickUnlink" title="Remove">
```

Comment:
Why do you need a div into a div as opposed to previous code which only had one?